Makefile

```diff
@@ -16,3 +16,8 @@ jahoda:
 
 test:
 	docker run --rm -it -v ${PWD}:/src -w /src $(IMAGE_NAME) ./bootstrap.sh
+
+check-bootstrap:
+	docker run --rm -it -v ${PWD}:/src -w /src $(IMAGE_NAME) bash -c ' \
```

Comment:
Thanks. I had to also add `Z` flag to the mount, i.e. `-v ${PWD}:/src:Z` to make it work on my machine (F28+enforcing selinux)

Reply:
it's probably time for me to enforce again